src/sentry/issues/escalating/escalating.py

```diff
@@ -277,14 +292,98 @@ def get_group_hourly_count(group: Group) -> int:
             raw_snql_query(request, referrer=IS_ESCALATING_REFERRER)["data"][0]["count()"]
         )
         cache.set(key, hourly_count, GROUP_HOURLY_COUNT_TTL)
+
+    return int(hourly_count)
+
+
+# TODO: probably some stuff to abstract out into EAP utils between here and src/sentry/eventstream/eap.py (and likely other read paths as well)
```

Comment:
Should we wait until we've worked on a couple more read paths to consolidate this logic? I feel like it's hard at this point to say exactly what kind of abstraction we want.